kymatio/scattering2d/tests/test_torch_scattering2d.py

```diff
@@ -0,0 +1,376 @@
+""" This script will test the submodules used by the scattering module"""
+
+import os
+import numpy as np
+import torch
+import pytest
+from kymatio.scattering2d import Scattering2D
+from torch.autograd import gradcheck
+from kymatio.backend.fake_backend import backend as fake_backend
+
+backends = []
+
+try:
```

Comment:
"These lines are repeated all over the place and should be defined as a function somewhere central." Comment by @janden .

Reply:
I kind of disagree, those lines are specific to a frontend... it's repeated .. 3 times!

Reply:
I see. Good point. Maybe opening an issue is better?

Reply:
(better than fixing it now)

Reply:
Done.